engine/src/cubos/engine/gizmos/types/box_gizmo.hpp

```diff
@@ -0,0 +1,58 @@
+#pragma once
+
+#include <glm/gtc/matrix_transform.hpp>
+
+#include <cubos/engine/gizmos/gizmos.hpp>
+
+/// @brief A gizmo that is a filled box
+class BoxGizmo : public cubos::engine::Gizmos::Gizmo
+{
```

Comment:
Even though this is a private header, you should still put this inside the namespace I think1